src/main/resources/avro/alleleAnnotationmethods.avdl

```diff
@@ -134,16 +134,9 @@ This request maps to the body of `POST /variantannotationsets/search` as JSON
 record SearchVariantAnnotationSetsRequest {
 
   /**
-  If non empty, will restrict the query to variant annotation sets within the
-  given dataset.
+  Restrict the query to variant annotation sets within this variant set.
```

Comment:
This should probably be "The VariantSet to search. " for consistency with the comments for other queries.